frontend/src/components/AuthManager/AuthManager.tsx

```diff
@@ -169,22 +174,37 @@ export const AuthManager = () => {
     />
   );
 
-  const SiteContent = () => (
-    <AuthContext.Provider value={{ logout, id, user, refreshUser }}>
-      <ReqContext.Provider value={{ withDefaults }}>
-        <SubscribeWrapper userId={id}>
-          <Switch>
-            <Route exact path="/" component={LoginPage} />
-            <PrivateRoute path="/admin" component={AdminRoutes} />
-            <PrivateRoute forRider path="/rider" component={RiderRoutes} />
-            <Route path="*">
-              <Redirect to="/" />
-            </Route>
-          </Switch>
-        </SubscribeWrapper>
-      </ReqContext.Provider>
-    </AuthContext.Provider>
-  );
+  const SiteContent = () => {
+    const { visible, message } = useToast();
+
+    return (
+      <>
+        {visible &&
+          createPortal(
+            <div>
+              <img alt="toast check" src={check} />
+              <p>{message}</p>
+            </div>,
+            document.body
+          )}
```

Comment:
can place this inside <Switch> below all the routes to avoid using a react fragment tag